goes16-l2-cmi.json

```diff
@@ -0,0 +1,569 @@
+    "temporal_composition_schema": null,
+    "composition_function": null,
+    "collection_type": "collection",
+    "metadata": {
+        "providers": [
+            {
+                "url": "https://www.nasa.gov/content/goes",
+                "name": "National Aeronautics and Space Administration (NASA)",
+                "roles": [
+                    "producer"
+                ]
+            },
+            {
+                "url": "https://www.goes-r.gov/",
+                "name": "National Oceanic and Atmospheric Administration (NOAA)",
+                "roles": [
+                    "processor",
+                    "producer",
+                    "licensor"
+                ]
+            },
+            {
+                "url": "https://data.inpe.br/big/",
+                "name": "National Institute for Space Research (INPE)",
+                "roles": [
+                    "host",
+                    "processor"
+                ]
+            }
+        ],
+        "license": {
+            "type": "Public Domain",
+            "uri": "https://www.nesdisia.noaa.gov/policy.html"
+        }
+    },
+    "keywords": [
+        "goes",
+        "goes-16",
+        "level-2",
+        "cloud",
+        "moisture",
+        "meteorology",
+        "atmosphere",
+        "surface",
+        "brazil"
+    ],
+    "is_public": true,
+    "is_available": true,
+    "category": "eo",
+    "version": 1,
+    "version_predecessor": null,
+    "version_successor": null,
+    "extent": {
+        "spatial": {
+            "bbox": [
+            [
+              -180,
+              -81.33,
+              6.3,
+              81.33
+            ],
+            [
+              -116.0,
+              -56.0,
+              -25.0,
+              35.0
+            ]
+          ]
+        }
+    },
+    "summaries": {
+        "platform": [
+            "GOES-16"
+        ],
+        "constellation": [
+            "GOES-R Series"
+        ],
+        "instruments": [
+            "ABI"
+        ]
+    },
+    "item_assets": {
+        "CMIP-C01-fdsk-nc": {
+            "type": "application/netcdf",
+            "title": "Cloud and Moisture Imagery reflectance factor - Band 01",
+            "roles": [
+                "data"
+            ],
+            "bands": [
```

Comment:
bands definition should be outside of item_assets and use the same key/values, feel free to use `null` for the attributes that aren't compatible with this kind of data.

Reply:
This is a question: How can I have two distinct `item_asset` items that refer to the same band?
Example:
```
"bands": [
       {
           "name": "ABI Band 1",
           "common_name": "blue",
           "description": "Daytime aerosol over land, coastal water mapping",
           "center_wavelength": 0.47
       }
]
(...)
"item_assets": {
        "CMIP-C01-fdsk-nc": { ==> This asset is related wit "ABI Band 1".
            "type": "application/netcdf",
            "title": "Cloud and Moisture Imagery reflectance factor - Band 01",
            "roles": ["data"]
        },
        "CMIP-C01-remap-nc": { ==> This asset is related wit "ABI Band 1" too.
            "type": "application/netcdf",
            "title": "Cloud and Moisture Imagery reflectance factor - Band 01 (EPSG:4326)",
            "roles": ["data"]
        }
}
````
For this reason, I included `bands` inside `item_assets`.